### PR TITLE
Azure UPI cluster in disconnected env

### PR DESCRIPTION
**Pull request by Amoghrd**

### CODE REVIEW

File: ci-operator/step-registry/cucushift/installer/rehearse/azure/upi/disconnected/provision/cucushift-installer-rehearse-azure-upi-disconnected-provision-chain.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  - chain: azure-provision-bastionhost
+  - ref: openshift-cluster-bot-rbac
+  - chain: mirror-images-by-oc-adm
+  - ref: ipi-conf
+  - ref: ipi-conf-mirror
+  - ref: ipi-conf-azure
+  - ref: ipi-conf-azure-provisioned-resourcegroup
+  - ref: ipi-conf-azure-provisionednetwork
+  - ref: ipi-install-monitoringpvc
+  - ref: upi-install-azure
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/rehearse/azure/upi/disconnected/provision/cucushift-installer-rehearse-azure-upi-disconnected-provision-chain.yaml:15]: Pls also append `  - ref: mirror-images-enable-qe-catalogsource` here.

**Comment by Amoghrd** [comment]
Rehearse passed
/assign @jianlinliu

**Pull request merged**